packages/Button/src/Button.styles.js

```diff
@@ -147,7 +146,7 @@ const sizeStyles = {
 // Kinds
 
 const kindStyles = props => ({
-  [Kinds.DEFAULT]: css`
+  [ButtonKinds.DEFAULT]: css`
```

Comment:
using my previous example this could be rewrite as:

```js
import { button } from "@paprika/buttons"
const { kind } = button;
const kindStyles = props => ({
  [kind.DEFAULT]: css``,
  [kind.PRIMARY]: css``,
  {...etc}
})
```